### PR TITLE
Create simp.c

### LINKED ISSUE #1145: SIMPSON’S RULE in c
I would like to do that. GSSOC19

### PR DESCRIPTION
**Pull request by gourabk121**
Fixes #1145

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- simpsons rule

#### Languages Used:
- c 

#### Files Added:
- simp.c

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by gourabk121** [comment]
@MastersAbh  review it plz

File: Simpsons_rule/simp.c [hunk 1]
```diff
@@ -0,0 +1,44 @@
+
+// Girlscript Summer Of Code
+// Simpson's rule  in c
+//this is used to find integration
```
Comment (MastersAbh) [inline Simpsons_rule/simp.c]: Please remove Girlscript comment and instead give description of problem statement.

File: Simpsons_rule/simp.c [hunk 2]
```diff
@@ -0,0 +1,44 @@
+
+// Girlscript Summer Of Code
+// Simpson's rule  in c
+//this is used to find integration 
+
+ #include<stdio.h>
```
Comment (MastersAbh) [inline Simpsons_rule/simp.c]: indentation

File: Simpsons_rule/simp.c [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+#include<conio.h>
+#include<math.h>
+
+ float f(float x);
```
Comment (MastersAbh) [inline Simpsons_rule/simp.c]: indentation

File: Simpsons_rule/simp.c [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+float f(float x) {
+    return (x);
+}
+int main() {
```
Comment (MastersAbh) [inline Simpsons_rule/simp.c]: blank line before main

File: Simpsons_rule/simp.c [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+    int a, b, n, i;
+    float s1 = 0, s2 = 0, sum, h;
+    printf("Enter the value of upper limit = ");
+    scanf("%d", & b);
+    printf("Enter the value of lower limit = ");
+    scanf("%d", & a);
+    printf("Enter the number of intervals = ");
+    scanf("%d", & n);
```
Comment (MastersAbh) [inline Simpsons_rule/simp.c]: no space after &

File: Simpsons_rule/simp.c [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+        }
+        sum = (h / 3) * (f(a) + f(b) + 4 * s2 + 2 * s1);
+        printf("the value is=%f", sum);
+    } else {
+        printf("Not appliciable! Interval must be even number");
+    }
+    getch();
+}
+
+ //output
```
Comment (MastersAbh) [inline Simpsons_rule/simp.c]: indentation

**Comment by gourabk121** [comment]
@MastersAbh  is it okay now?

**Comment by gourabk121** [comment]
@rishabhgarg25699  how to squash

**Comment by MastersAbh** [comment]
Hi @gourabk121 please connect to me on Slack. I'd be happy to help.

**Comment by MastersAbh** [comment]
@gourabk121 you have 30 mins. Are you available ?

**Comment by gourabk121** [comment]
yeah I am

**Pull request merged**